spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/Blob.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intel.analytics.bigdl.nn.mkldnn
+
+import com.intel.analytics.bigdl.tensor.{DnnTensor, Tensor}
+
+private[mkldnn] class Blob(private val _size: Array[Int]) extends Serializable {
+  val dense: Tensor[Float] = Tensor[Float](_size)
+  val native: DnnTensor[Float] = DnnTensor[Float](_size)
+
+  private var _shape: MemoryData = _
+
+  def sync(isDense2Native: Boolean = true): Unit = {
```

Comment:
I suggest you use two API instead of one API with a flag

Reply:
OK.